frontend/simple/main.js

```diff
@@ -10,54 +11,91 @@ import utils, { wrap, lazyLoadVue, superagentHeader } from './js/utils'
 import store from './js/state'
 import './js/transitions'
 
-Vue.config.debug = process.env.NODE_ENV === 'development'
 Vue.use(Router)
-Vue.use(require('vue-form'), {invalidClass: 'is-danger'})
+Vue.use(VeeValidate)
```

Comment:
Curious, is the recently announced [Vuelidate](http://monterail.com/blog/2016/rethinking-validations-for-vue-js/) worth looking into? I prefer whichever is simpler.